src/DGSM.jl

```diff
@@ -18,8 +24,11 @@ The inputs to the function DGSM are as follows:
     Array of distribution of respective variables
     Eg- dist = [Normal(5,6),Uniform(2,3)]
 	for two variables
+4.crossed:
+    A string(True/False) which act as indicator for computation of DGSM crossed indices
+    Eg- a True value over there will lead to evauation of crossed indices
 """
-function DGSM(f,samples::Int,distr::AbstractArray)
+function DGSM(f,samples::Int64,distr::AbstractArray, crossed::String)
```

Comment:
crossed=false should be a keyword argument and a boolean.

Reply:
samples should be an `Int`

Reply:
I have made the above changes